Modules/Office365DSC/DSCResources/MSFT_O365User/MSFT_O365User.psm1

```diff
@@ -171,7 +171,7 @@ function Get-TargetResource
     }
     catch
     {
-        Write-Verbose -Message "The specified User doesn't already exist."
+        $Message = "The specified User {$UserPrincipalName} doesn't already exist."
```

Comment:
Nothing is done with this Message variable.

Reply:
Good Catch. Fixed